README.md

```diff
@@ -42,6 +40,10 @@ If you wish to manually create and publish package for Python:
 
 ## Changes
 
+### 3.0.3
+
++ Up version th2-grpc-common to 3.8.0
```

Comment:
Could we synchronize java and python versions?

Reply:
They're already synchronized. Both Java and Python have version 3.0.3 in these changes.